### ISSUE TITLE
self in self.crypto

### ISSUE THREAD

**Issue by subzey** [ai/nanoid#189]
Getting rid of `self.` in `self.crypto` may shave a few bytes for index.browser.js

```diff
-let random = bytes => self.crypto.getRandomValues(new Uint8Array(bytes))
+let random = bytes => crypto.getRandomValues(new Uint8Array(bytes))
```

Previously it was used to safely access `self.msCrypto` if `self.crypto` is undefined, but now it is useless:
- If there is `[[global]].crypto`, it's okay either way.
- If there's none, one way we get a `ReferenceError` the othe way we get a `TypeError`. The outcome is essentially the same.

----

Sorry for not providing concrete figures, but the `npx size-limit` / `npx yarn size-limit` doesn't work for me, at least on Win10 and Win10 WSL

**Comment by ai** [comment]
1. Did you liked our Gitpod page for quick hacks? https://gitpod.io/#https://github.com/ai/nanoid/
2. What error did you have? Seems like I never to fix Size Limit

**Comment by ai** [comment]
Fixed 4c3e635

**Comment by subzey** [comment]
Sorry for my late answer.

1. It looks nice, but isn't it just a cloud IDE? I've got a regular one already. 🤷‍♀️

2. It behaves differently:

On "plain old" Win 10 the size-limit complains it cannot spawn `npx`. I guess the reason is that its executable is named `npx.cmd` and [there are known gotchas](https://nodejs.org/api/child_process.html#child_process_spawning_bat_and_cmd_files_on_windows).

On Win10/WSL it tells me to give some coverage file the other name. Probably It will be fixed with some kind of `git clean` or something. I'll tell later if you don't mind

**Comment by subzey** [comment]
Update: After some `git clean`s in WSL env it works as a charm.

Plain Win10 has errors
`eslint-ci` reports this:
```
...\nanoid\test\demo\index.js
4:1  error  There should be no empty line within import group  import/order
...\nanoid\test\update-prebuild.js
3:21  error  There should be at least one empty line between import groups  import/order
4:18  error  There should be at least one empty line between import groups  import/order
4:18  error  `terser` import should occur after import of `fs`              import/order 
```
I'm not sure it's false positive or a POSIX env returns false negative.

----
`yarn size-limit`:
```
× Compiling files to ESM
ERROR  Error: spawn npx ENOENT
at Process.ChildProcess._handle.onexit (internal/child_process.js:264:19)
at onErrorNT (internal/child_process.js:456:16)
at processTicksAndRejections (internal/process/task_queues.js:77:11)
error Command failed with exit code 1.
```

I actually run it as `npx yarn size-limit` _(I'm not a huge fan of installing yarn or gulp or whatever globally)_ so I'm sure `npx` is installed.

**Comment by ai** [comment]
Can you open `node_modules/@size-limit/dual-publish/index.js` and replace

```js
    let cmd = spawn(
      'npx', ['-q', 'dual-publish', '--check'], {
        cwd: config.cwd
      }
    )
```

to

```js
    let cmd = spawn(
      process.platform === 'win32' ? 'npx.cmd' : 'npx',
      ['-q', 'dual-publish', '--check'],
      { cwd: config.cwd }
    )
```

Will it help?

**Comment by subzey** [comment]
Yup, it fixes this issue, but there's one more.

It throws 

```
× Adding to empty webpack project                                                                               ERROR  Error:                                                                                                 ERROR in C:/Users/subze/AppData/Local/Temp/size-limit-stz0OkINoRFKpDQxh702w/entry.js                           Module not found: Error: Can't resolve... 
```
I've found out the reason is (tmpdir)/.../entry.js
```
import { customAlphabet } from 'C:\Users\subzey\Code\nanoid\dual-publish-tmp\async\index.js'
console.log(customAlphabet)
```

Please note the unescaped backslashes!

**Comment by subzey** [comment]
node_modules/@size-limit/webpack/get-config.js

```diff
       let list = check.import[i].replace(/}|{/g, '').trim()
-      loader += `import ${ check.import[i] } from '${ i }'\n` +
+      loader += `import ${ check.import[i] } from ${ JSON.stringify(i) }\n` +
                 `console.log(${ list })\n`
```
fixes this issue

**Comment by ai** [comment]
Pushed with your name https://github.com/ai/size-limit/commit/a0adb2dfd597e7dafe10c5cfe4317f1bad3d1881

Released at 4.4.3

**Comment by subzey** [comment]
Haven't you update the https://github.com/ai/size-limit/blob/master/packages/dual-publish/index.js#L14?

**Comment by ai** [comment]
Oops. `npx` released in 4.4.5

**Comment by subzey** [comment]
I confirm, it works now!

(Though I had to remove `yarn.lock` in this project first to update to the newest version)
